File: Swift/Utils/Utils/JailbreakUtils.m
#import "JailbreakUtils.h"

#define JAILBREAK_FILES @[@"/Application/Cydia.app",@"/Application/limera1n.app",@"/Application/greenpois0n.app",@"/Application/blackra1n.app",@"/Application/blacksn0w.app",@"/application/redsn0w.app"]

@implementation JailbreakUtils

+(BOOL) isJailbreaked {
    BOOL ret = NO;
    NSFileManager * fmgr = [NSFileManager defaultManager];
    for (NSString * file in JAILBREAK_FILES) {
        if ([fmgr fileExistsAtPath:file]) {
            ret = YES;
            break;
        }
    }
    return ret;
}


@end

File: Swift/Utils/JailbreakUtils.swift
import UIKit

class JailbreakUtils: NSObject {
    class func isJailbreaked() -> Bool {
        var ret = false
        var jailBreakFiles = [
            "/Application/Cydia.app",
            "/Application/limera1n.app",
            "/Application/greenpois0n.app",
            "/Application/blackra1n.app",
            "/Application/blacksn0w.app",
            "/application/redsn0w.app"
        ]
        var fmgr = NSFileManager.defaultManager()
        for file in jailBreakFiles {
            if (fmgr.fileExistsAtPath(file)) {
                ret = true
                break
            }
        }
        return ret
    }
}
